Popup window

sunga12 opened the pull request:
Add javascript for featured project and popup window

sunga12 committed:
Add javascript for featured project and popup window

sumon766 committed:
Add script for recent works section

sunga12 committed:
Re-add featured work image

sunga12 committed:
Fix linter errors

sunga12 committed:
Fix webhint error

omar-muhamad reviewed (commented):
# Invalid ❌

Hi Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] Eslint linters GitHub action is not configured, Please use this [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) file instead of the one you used to make it work because for this project you suppose to add linters actions for [HTML, CSS, and javascript.](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

sunga12 committed:
Use Correct Linters File

GzimAsani commented on script.js:
```diff
@@ -18,4 +18,336 @@ lists.forEach((list) => {
+    popupTitle.appendChild(popupClose);
+
+    const popupLangs = document.createElement('ul');
+    popupLangs.className = 'popup-buttons';
```
- [ ] Kindly adjust the padding of buttons in order to match the figma design

Yours:
![image](https://user-images.githubusercontent.com/62705947/217575179-2a2aa62c-4a7b-4e35-86af-683c5fbfe889.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/217575242-d1e165ae-3e1f-4c90-b4fd-04ef7a8c53b6.png)

GzimAsani replied:
- [ ] Add paddings on the side and adjust the X button, check the images for more info

Yours:
![image](https://user-images.githubusercontent.com/62705947/217575917-81e89257-260c-451a-bb7a-efb336d2c0cd.png)

Design:
![image](https://user-images.githubusercontent.com/62705947/217576005-cfd98382-ceed-4219-b5a0-60ec888830ff.png)

mckent05 replied:
- Good job done so far, but as a result of the adjustments made following the last review, you seem to have tampered with your live buttons, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="812" alt="ergergaerg" src="https://user-images.githubusercontent.com/73607512/217650931-bf7f67a2-da60-4129-9d02-179098bd6ea1.PNG">

Figma design:
<img width="492" alt="q3wtgq34t" src="https://user-images.githubusercontent.com/73607512/217651019-3bd23e04-4524-4ab7-b218-13c304b4c20a.PNG">
- Also, as a result of the adjustments you made following the last review, you seem to have introduced a bug. Please fix this 🙏 
 Your project:
<img width="265" alt="q34tq34t" src="https://user-images.githubusercontent.com/73607512/217651518-a103d34e-a624-493f-a844-bcb7beb95904.PNG">

Design template:
<img width="147" alt="efQEFqe" src="https://user-images.githubusercontent.com/73607512/217651653-9e267d1e-0993-4d99-9377-4864952c383c.PNG">

GzimAsani commented on script.js:
```diff
@@ -18,4 +18,336 @@ lists.forEach((list) => {
+  popupTitle.appendChild(popupClose);
+
+  const popupLangs = document.createElement('ul');
+  popupLangs.className = 'popup-buttons';
```
- [ ] Please implement styling for buttons, hover and pressed 
![image](https://user-images.githubusercontent.com/62705947/217575487-b4be14de-7d16-4358-9261-9fdcac9b4a15.png)

GzimAsani reviewed (changes requested):
### Hello @sunga12                     :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with adding javascript, it looks great :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 


 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sunga12 committed:
Update based on 1st Code Review

sunga12 commented:
Changes Required:

- Adjust the padding of the buttons in order to match the Figma design
- Add paddings on the side of the Image in the mobile popup window and adjust the X button according to Figma design
- Implement styling for buttons: hover, and pressed

Changes Made:

- Adjusted the padding of the buttons in order to match the Figma design
- Added paddings on the side of the Image in the mobile popup window and adjust the X button according to Figma design
- Implemented styling for buttons: hover and pressed

mckent05 reviewed (changes requested):
Hi @sunga12, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![almost there](https://user-images.githubusercontent.com/73607512/207265397-5ef597f5-a50c-49b4-8f07-7548b0520e98.gif)


 ## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sunga12 committed:
Update according to 2nd code review

sunga12 commented:
Changes made:

- Fixed bugs related to live buttons in popup window.

m05e5 reviewed (approved):
**Hi @sunga12 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![CleverScornfulBuffalo-size_restricted](https://user-images.githubusercontent.com/37901469/208869296-5ca9f6cc-a1b5-48fd-bd59-78fd83b35471.gif)

## Highlights

✔️ Well done implementing the requested changes 

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
